sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -129,7 +129,16 @@ case class DescribeFunction(
             result
           }
 
-        case None => Seq(Row(s"Function: $functionName not found."))
+        case None =>
+          val currentDb = sparkSession.sessionState.catalog.getCurrentDatabase
+          if (sparkSession.externalCatalog.functionExists(currentDb, functionName)) {
+            val catalogFunction = sparkSession.externalCatalog.getFunction(currentDb, functionName)
+            Row(s"Function: ${catalogFunction.identifier}") ::
+              Row(s"Class: ${catalogFunction.className}") ::
+              Row(s"Usage: N/A") :: Nil
+          } else {
+            Seq(Row(s"Function: $functionName not found."))
+          }
```

Comment:
We also need to change the parser and change the interface of `DescribeFunction` to:
```scala
case class DescribeFunction(
    functionName: FunctionIdentifier,
    isExtended: Boolean) extends RunnableCommand {
```